modules/virt-hot-unplugging-bridge-network-interface-cli.adoc

```diff
@@ -16,12 +21,35 @@ You can remove a bridge network interface from a running virtual machine (VM).
 
 .Procedure
 
-. Hot unplug a bridge network interface by running the following command. The `virtctl removeinterface` command detaches the network interface from the guest, but the interface still exists in the pod.
+. Hot unplug a secondary network interface by setting the interface state to `absent` in the VM specification. This detaches the network interface from the guest, but the interface still exists in the pod.
 +
-[source,terminal]
+.Example VM configuration
+[source,yaml]
 ----
-$ virtctl removeinterface <vm_name> --name <interface_name>
+apiVersion: kubevirt.io/v1
+kind: VirtualMachine
+metadata:
+  name: vm-fedora
+template:
+  spec:
+    domain:
+      devices:
+        interfaces:
+          - name: defaultnetwork
+            masquerade: {}
+          # set the interface state to absent 
+          - name: <secondary_nic>
+            state: absent <1>
+            bridge: {}
+    networks:
+      - name: defaultnetwork
+        pod: {}
+      - name: <secondary_nic>
+        multus:
+          networkName: <nad_name>
+# ...
 ----
+<1> Set the interface state to `absent`.
```

Comment:
I would also mention that removing the interface from the spec wouldn't cause hot-unplug.
`Set the interface state to `absent` (removing the interface from the spec wouldn't cause hot-unplug)`

Reply:
@AlonaKaplan In step 1 of this procedure, I have mentioned "This detaches the network interface from the guest, but the interface still exists in the pod." Do you want me to add that sentence to the callout <1> description as well?

Reply:
I just want to emphasize that only marking the interface as absent will detach the interface from a running guest.
Removing the interface completely for the spec won't do it.